.github/workflows/create-release-pull-request.yaml

```diff
@@ -0,0 +1,41 @@
+name: Create release pull request
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersionNumber:
+        description: 'New version number'
+        required: true
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: fregante/setup-git-user@v1
```

Comment:
You no longer need this. `peter-evans/create-pull-request` takes care of it all for you.

```suggestion
```